packages/cloud/src/upload/createUploadStore.ts

```diff
@@ -0,0 +1,39 @@
+import create from 'zustand';
+import { Upload, UploadState } from './types';
+
+/**
+ * Creates an origin store using `zustand`.
+ *
+ * The purpose of this is to keep track of uploads and their progress but only
+ * storing the key to the lookup in the Element itself. We do it this way
+ * because we don't want to modify the Editor value during the upload or it
+ * becomes part of the edit history.
+ */
+export const createUploadStore = (
```

Comment:
Let's use `createStore` exported by `@udecode/plate-core`, it's a utility I've created for an easier API

Reply:
See https://github.com/udecode/zustood

Reply:
Updated createStore and associated code to use zustood instead of zustand. Done.